chemicalx/models/deepdrug.py

```diff
@@ -1,14 +1,102 @@
 """An implementation of the DeepDrug model."""
 
-from .base import UnimplementedModel
+from typing import Optional
+
+import torch
+from torchdrug.data import PackedGraph
+from torchdrug.layers import GraphConv, MaxReadout
+
+from chemicalx.constants import TORCHDRUG_NODE_FEATURES
+from chemicalx.data import DrugPairBatch
+from chemicalx.models import Model
 
 __all__ = [
     "DeepDrug",
 ]
 
 
-class DeepDrug(UnimplementedModel):
+class DeepDrug(Model):
     """An implementation of the DeepDrug model.
 
-    .. seealso:: https://github.com/AstraZeneca/chemicalx/issues/14
+    .. based on:: https://www.biorxiv.org/content/biorxiv/early/2020/11/10/2020.11.09.375626.full.pdf
```

Comment:
please put an actual citation

Reply:
`seealso` is also a special directive in RST, not sure why you changed it to `based on` which causes a parse error